docs/en/observability/monitor-k8s/monitor-k8s-logs.asciidoc

```diff
@@ -246,7 +246,7 @@ processors:
 . If you're running {filebeat} on master nodes, check to see if the nodes use
 https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints].
 Taints limit the workloads that can run on master nodes. If necessary, update
-the DaemonSet spec to include tolerations:
+the DaemonSet spec to include `tolerations`:
```

Comment:
I wouldn't put this in code formatting because tolerations are a K8s concept. I would just override the linter here.